### PR TITLE
[connectivity] update the connectivity readme to include info about changes in iOS 13

### PR DESCRIPTION
**Pull request by danwulff**
## Description

Update the Readme for the Connectivity plugin to include information about changes in iOS 13.

Specifically the tightening of security around SSID and BSSID information.

Apple's email announcing the change:
image

Apple's documentation about [CNCopyCurrentNetworkInfo](https://developer.apple.com/documentation/systemconfiguration/1614126-cncopycurrentnetworkinfo)

## Related Issues

Resolves: [#35322](https://github.com/flutter/flutter/issues/35332)

A followup issue has been created for "fixing" the iOS 13 functionality at [#37804](https://github.com/flutter/flutter/issues/37804)

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [ X ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ X ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ X ] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [ ] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [ ] I updated CHANGELOG.md to add a description of the change.
- [ X ] I signed the [CLA].
- [ X ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [ X ] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by danwulff** [comment]
@cyanglaz Do I also need to increment the package version and add to the changelog, since it's just a readme change?

File: packages/connectivity/README.md [hunk 1]
```diff
@@ -48,6 +50,31 @@ dispose() {
...
+}
+```
+
+### Known Issues
+
+#### iOS 13
+
+The methods `.getWifiBSSID()` and `.getWifiName()` utilize the [CNCopyCurrentNetworkInfo](https://developer.apple.com/documentation/systemconfiguration/1614126-cncopycurrentnetworkinfo) function on iOS.
+
+As of iOS 13 that api will no longer return valid information by default and will instead return the following:
```
Comment (cyanglaz) [inline packages/connectivity/README.md]: nits:
As of iOS 13, these APIs..

File: packages/connectivity/README.md [hunk 2]
```diff
@@ -48,6 +50,31 @@ dispose() {
...
+> SSID: "Wi-Fi" or "WLAN" ("WLAN" will be returned for the China SKU)  
+> BSSID: "00:00:00:00:00:00"
+
+You can follow issue [#37804](https://github.com/flutter/flutter/issues/37804) for the changes required to return valid SSID and BSSID values with iOS 13.
```
Comment (cyanglaz) [inline packages/connectivity/README.md]: We should just explain how can these work in detail. like what was explained from Apple's email.
And I think one of the options to make this work is to request user's location service permission. So let's add that to the doc as well. 
You can find an example here in the image picker plugin's readme asking developer to put necessary keys into the info.plist.
https://github.com/flutter/plugins/blob/master/packages/image_picker/README.md

Comment (cyanglaz) [reply packages/connectivity/README.md]: >And I think one of the options to make this work is to request user's location service ?>permission. So let's add that to the doc as well.
>You can find an example here in the image picker plugin's readme asking developer to >put necessary keys into the info.plist.
>https://github.com/flutter/plugins/blob/master/packages/image_picker/README.md

I think we should do this with the PR that resolves https://github.com/flutter/flutter/issues/37804. So nothing needs to be done here.

**Comment by cyanglaz** [comment]
@danwulff Also, I think the fix you mentioned in https://github.com/flutter/flutter/issues/37804 with the location request is pretty straightforward. If you want to include the fix in a same PR I'm happy to review it too.

**Comment by danwulff** [comment]
@cyanglaz I would love to make the code changes as well, but I don't have the ability to test with iOS 13 currently.

Will make changes per your suggestions by end of day.

File: packages/connectivity/README.md (hunk 2 above)
Comment (danwulff) [reply packages/connectivity/README.md]: Won't be done now. Resolving

**Comment by cyanglaz** [comment]
@danwulff That's great! Thanks! There is no rush, please take your time :)

File: packages/connectivity/README.md (hunk 1 above)
Comment (danwulff) [reply packages/connectivity/README.md]: updated

**Comment by danwulff** [comment]
@cyanglaz Let me know if you see any other changes that need to happen.

**Comment by cyanglaz** [comment]
@danwulff Can we include the fixes in this same PR?

**Comment by danwulff** [comment]
@cyanglaz I can't sorry. I don't have the capability of using Xcode 11 (iOS 13 SDK).

If you or another googler want to add on to this PR feel free.

**Comment by cyanglaz** [comment]
@danwulff Oh, sorry i didn't read your last comment clearly. That's ok. 

> @cyanglaz I can't sorry. I don't have the capability of using Xcode 11 (iOS 13 SDK).
> 
> If you or another googler want to add on to this PR feel free.

Looks good! Going to merge when green.

**Pull request merged**